apps/library/lib/library/game.ex

```diff
@@ -112,7 +120,8 @@ defmodule Library.Game do
       %{
         game_state |
           board: Board.generate,
-          word_list: []
+          wordlist: [],
+          game_over: false
       }
     }
```

Comment:
Just a though, What if we define "initalState" as a module attr then just reference that here - kind of how we do in elm